Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1156,17 +1156,34 @@ def appendContext(key, data, dedup=False):
     if data is None:
         return
     existing = demisto.get(demisto.context(), key)
+
     if existing:
-        strBased = isinstance(data, STRING_TYPES) and isinstance(existing, STRING_TYPES)
-        if strBased:
-            data = data.split(',')
-            existing = existing.split(',')
-        newVal = data + existing
-        if dedup:
-            newVal = list(set(newVal))
-        if strBased:
-            newVal = ','.join(newVal)
-        demisto.setContext(key, newVal)
+        if isinstance(existing, STRING_TYPES):
+            if isinstance(data, STRING_TYPES):
+                new_val = data + ',' + existing
```

Comment:
let's create a list of values here

Reply:
Are you sure? 
![image](https://user-images.githubusercontent.com/50294648/81498674-f2a46600-92ce-11ea-83fd-ff3e02a30305.png)
It's breaking the original behavior

Reply:
if it's breaking then no need